### PR TITLE
fix(wallet): fix inputs selection with the change config

### PR DESCRIPTION
**Pull request by MSzalowski**
## List of changes

- fix inputs selection with the change config

Based on the @Ryun1 change [here](https://github.com/Ryun1/cip95-cardano-wallet-connector/pull/80/files) :tada:

## Checklist

- [x] My changes generate no new warnings
- [x] My code follows the [style guidelines](https://github.com/IntersectMBO/govtool/tree/main/docs/style-guides) of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the [changelog](https://github.com/IntersectMBO/govtool/blob/main/CHANGELOG.md)
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by Ryun1** [comment]
Wait!

The fix wasnt done yet, I need to test it out

**Comment by Ryun1** [comment]
has anyone ever got the `InputSetEmptyUTxO` via GovTool? 🤔

**Comment by Ryun1** [comment]
You can remove the old  `txBuilder.add_change_if_needed(shelleyChangeAddress);`

see https://github.com/IntersectMBO/govtool/pull/1615

**Comment by Ryun1** [comment]
I have now tested using this setup and it looks good to me!

**Review by Ryun1 — APPROVED** [review]

**Comment by MSzalowski** [comment]
We have the working dev environment (except one known Issue), I'll merge that changes to be included in the v1.0.9. Thanks once again for your help @Ryun1 :muscle:

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ changes.
 
 - Fix querying metadata-validation service from haskell backend [Issue 1612](https://github.com/IntersectMBO/govtool/issues/1612)
 - Fix thread killed by timeout manager [Issue 1417](https://github.com/IntersectMBO/govtool/issues/1417)
+- Fix `InputSetEmptyUTxO` error
```
Comment (Ryun1) [inline CHANGELOG.md:22]:
```suggestion
- Changed use new CSL UTxO selection to take into account change
```

**Pull request merged**